discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -337,3 +358,73 @@ def setup_nodes(self, nodes):
         nodes.start_all()
         self.wait_until_hosts_are_discovered(nodes_count=len(nodes))
         return nodes.create_nodes_cluster_hosts_mapping(cluster=self)
+
+    def wait_for_cluster_validation(
```

Comment:
naming: wait_for_validation
wait_for_host_validation already differentiates 
wdty?

Reply:
Isn't it better to be explicit?  If someone saw wait_for_validation they might assume it's for both?

Reply:
okay, sure

Reply:
@avishayt what is cluster validation?